upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -1142,6 +1155,37 @@ else
   echo "====> ${state_name} has been completed" | tee -a "${LOG_FILE}"
 fi
 
+state_name="VALIDATE_SPIRE_POSTGRES_HEALTH"
+state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
+if [[ ${state_recorded} == "0" ]]; then
+  echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
+  {
+    # make sure spire-postgres is healthy
+    i=1
+    max_checks=40
+    wait_time=15
+    healthy="false"
+    while [[ ${i} -le ${max_checks} ]]; do
+      if [[ "$(kubectl get postgresql cray-nls-postgres -n argo -ojsonpath='{.status.PostgresClusterStatus}')" == "Running" ]]; then
```

Comment:
should this be 'kubectl get postgresql spire-postgres -n spire...' ??

Reply:
it should be. fixed.